sections/03_categorize.md

```diff
@@ -28,7 +28,148 @@ we share these data while respecting privacy? I've made a section for each of
 these. Feel free to add more. I see each section as something on the order of
 1-2 paragraphs in our context.*
 
-#### Standardization/integration
+#### Clinical Care
+
+#### Imaging Applications in Health Care
+
+One of the general areas where deep learning methods have had substantial
+success has been in image analysis. Applications in areas of medicine that use
+imaging extensively are also emerging. Mammography has been one area with
+numerous contributions [@doi:10.1007/978-3-319-46723-8_13,
+@doi:10.1007/978-3-319-24553-9_74, @doi:10.1101/095794, @doi:10.1101/095786]. In
+all of this work, the researchers must work around a specific challenge - the
+limited number of well annotated training images. To expand the number and
+diversity of images, the researchers have employed approaches where they employ
+adversarial examples [@doi:10.1101/095786] or first train towards human-created
+features before subsequent fine tuning [@doi:10.1007/978-3-319-46723-8_13]. The
+presence of a large bank of well-annotated mammography images would aid in the
+application of deep neural networks to this area. Though this strategy has not
+yet been employed in this domain, large collections of unlabeled images might
+first be used in an unsupervised context to construct high-quality feature
+detectors. Then the small number of labeled examples could be used for
+subsequent training. Similar strategies have been employed for EHR data where
+high-quality labeled examples are also difficult to obtain
+[@doi:10.1101/039800].
+
+In addition to radiographic images, histology slides are also being analyzed
+with deep learning approaches. In recent work, Wang et al.[@arxiv:1606.05718]
+analyzed stained slides to identify cancers within slides of lymph node slices.
+The approach provided a probability map for each slide. On this task a
+pathologist has about a 3% error rate. Their algorithm had about a 7% error
```

Comment:
Do we need to be more precise or is this the terminology used in the paper?  Does their error rate include false positives and false negatives?